Panic when listing feature after registering

Repro procedure:

```
$ featctl init
$ featctl register entity device --length 32
$ featctl register entity user --length 64 --description "all users"
$ featctl register group phone --entity device
$ featctl register batch-feature price --group phone --db-value-type "int"
$ featctl list feature
Name,Group,Entity,Category,DBValueType,ValueType,Description,OnlineRevision,OfflineLatestRevision,OfflineLatestDataTable,CreateTime,ModifyTime
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x58c0ed]

goroutine 1 [running]:
github.com/oom-ai/oomstore/pkg/oomstore/types.(*Feature).ToCsvRecord(0xc0000ac680)
        /home/wenxuan/develop/oom.ai/oomstore/pkg/oomstore/types/feature.go:103 +0x12d
github.com/oom-ai/oomstore/featctl/cmd.glob..func12(0xc7d820, {0x8dfc3e, 0x0, 0x0})
        /home/wenxuan/develop/oom.ai/oomstore/featctl/cmd/list_feature.go:38 +0x3e5
github.com/spf13/cobra.(*Command).execute(0xc7d820, {0xcb7b38, 0x0, 0x0})
        /home/wenxuan/.go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:860 +0x5f8
github.com/spf13/cobra.(*Command).ExecuteC(0xc7ec20)
        /home/wenxuan/.go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
        /home/wenxuan/.go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
github.com/oom-ai/oomstore/featctl/cmd.Execute()
        /home/wenxuan/develop/oom.ai/oomstore/featctl/cmd/root.go:35 +0x25
main.main()
        /home/wenxuan/develop/oom.ai/oomstore/featctl/main.go:6 +0x17
```

--- a/pkg/oomstore/types/feature.go
+++ b/pkg/oomstore/types/feature.go
@@ -99,7 +99,7 @@ func (rf *Feature) ToCsvRecord() string {
 	if rf.OfflineRevision != nil {
 		offlineRevision = fmt.Sprint(*rf.OfflineRevision)
 	}
-	if rf.OfflineDataTable == nil {
+	if rf.OfflineDataTable != nil {
 		offlineDataTable = *rf.OfflineDataTable
 	}
 

